kserve-rest-proxy.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: kserve-rest-proxy
+  version: 0.12.0
+  epoch: 0
+  description: "REST API proxy for kserve, the standardized serverless ML inference platform on Kubernetes"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - go
```

Comment:
We can drop the environment here. `go` is included as a part of `go/build` below

Reply:
TY TY, addressed